choreography/src/choreography/Fabrikam.Choreography.ChoreographyService/Controllers/ChoreographyController.cs

```diff
@@ -56,12 +57,12 @@ public async Task<IActionResult> Operation([FromBody] EventGridEvent[] events)
                 return BadRequest("No Event for Choreography");
             }
                 
-            if (events[0].EventType is EventTypes.EventGridSubscriptionValidationEvent)
+            if (events[0].EventType is "Microsoft.EventGrid.SubscriptionValidationEvent")
```

Comment:
This can use the constant - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs#L233

Reply:
Thanks!